hardware/raspberrypi/power/README.md

```diff
@@ -1,11 +1,39 @@
 # Power Supply
 
-The Raspberry Pi 3 is powered by a +5.1V micro USB supply. Exactly how much current (mA) the Raspberry Pi requires is dependent on what you connect to it. We have found that purchasing a 2.5A power supply from a reputable retailer will provide you with ample power to run your Raspberry Pi. You can purchase the [official Raspberry Pi Power Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) from our website, and you can learn more about the differing power requirements of the various models of the Raspberry Pi on our [FAQ page](https://www.raspberrypi.org/help/faqs/#power). 
+The power supply requirements differ by Raspberry Pi model. All models require a 5.1V supply, but the current supplied generally increases according to model. All models up to the Raspberry Pi 3 require a microUSB power connector, whilst the Raspberry Pi 4 uses a USB-C connector.
 
-Typically, the model B uses between 700-1000mA depending on what peripherals are connected; the model A can use as little as 500mA with no peripherals attached. The maximum power the Raspberry Pi can use is 1 Amp. If you need to connect a USB device that will take the power requirements above 1 Amp, then you must connect it to an externally-powered USB hub.
+Exactly how much current (mA) the Raspberry Pi requires is dependent on what you connect to it. The following table gives minimum and typical current requirements. 
+
+| Product | Recommended PSU current capacity | Maximum total USB peripheral current draw | Typical bare-board active current consumption |
+|-|-|-|-|
+|Raspberry Pi Model A | 700mA | 500mA | 200mA |
+| Raspberry Pi Model B |1.2A | 500mA | 500mA |
+| Raspberry Pi Model A+ | 700mA | 500mA | 180mA
+| Raspberry Pi Model B+ | 1.8A | 600mA/1.2A (switchable)| 330mA |
+| Raspberry Pi 2 Model B | 1.8A | 600mA/1.2A (switchable) | 350mA |
+| Raspberry Pi 3 Model B | 2.5A | 1.2A | 400mA |
+| Raspberry Pi 3 Model A+ | 2.5A | Limited by PSU, board, and connector ratings only. | 350mA |
+| Raspberry Pi 3 Model B+ | 2.5A | 1.2A | 500mA |
+| Raspberry Pi 4 Model B | 3.0A | 1.2A | 600mA |
+| Raspberry Pi Zero | 1.2A | Limited by PSU, board, and connector ratings only | 100mA |
+| Raspberry Pi Zero W/WH | 1.2A | Limited by PSU, board, and connector ratings only.| 150mA |
+
+Raspberry Pi have developed their own power supplies for use with all models. These are reliable, use heavy gauge wires and are reasonably priced. 
+
+For Raspberry Pi 0-3, we recommend our [micro USB Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/). For Raspberry Pi 4, we recommend our [USB-C Supply](https://www.raspberrypi.org/products/type-c-power-supply/)
```

Comment:
Given the "Recommended PSU current capacity" column in the table above, is it worth mentioning the current-capacity of the official PSUs here?

Reply:
Hmm, my first thought was the information is only a link away, but actually, why should the reader need to click a link. I'll add them.